Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM python:3.6
+
+RUN apt-get update && apt-get install -y postgresql-client
```

Comment:
Is this because the image comes with postgresql/postgis installed?

Reply:
It's so you can access the postgis in the other container in the network using psql, postgis/postgres isn't in this container though

Reply:
ah, I see, the `build: .` command is using this dockerfile... I just need to go learn more docker-compose stuff. It'll be good to get this set up properly on the server too. We're currently running everything in screens and it all went down today, I suspect due to server reboots.